core/src/main/java/org/elasticsearch/transport/netty/NettyTransport.java

```diff
@@ -225,7 +243,8 @@
     final ScheduledPing scheduledPing;
 
     @Inject
-    public NettyTransport(Settings settings, ThreadPool threadPool, NetworkService networkService, BigArrays bigArrays, Version version, NamedWriteableRegistry namedWriteableRegistry) {
+    public NettyTransport(Settings settings, ThreadPool threadPool, NetworkService networkService, BigArrays bigArrays, Version version,
+            NamedWriteableRegistry namedWriteableRegistry) {
```

Comment:
This is a different indent than the standard, Intellij and the .dir-locals.el indent these to be opening `(` for class constructors (personally I don't like it, but Intellij ಠ_ಠ)